runtime/onert/core/src/compiler/pass/ConstantInsertionPass.cc

```diff
@@ -19,6 +19,7 @@
 #include "backend/Backend.h"
 #include <ir/Graph.h>
 #include <util/Utils.h>
+#include "util/logging.h"
```

Comment:
@wateret I have question about coding style rule. Because 

i) line 21 uses `<` and line 22 uses `"` for headers under same folder "util" and 
ii) there is no grouping between `< >` headers and `" "` headers.

Do we have no rule for this two cases? I guess no rules because I cannot find the coding style document and the format checker does not complains. :)

Reply:
They all should be `""`, as they are in the same module with this source file. I'll fix those too.
And there is no rules for grouping(AFAIR), but I personally prefer grouping `<>`s and `""`. (But keep the header file for itself at the top)